bigrig/config.py

```diff
@@ -63,7 +66,7 @@
         "target": {
             "type": "object",
             "properties": {
-                "variables": {
+                "vars": {
```

Comment:
I'm not very keen on using build-in functions as object attributes.

Reply:
Sorry, I don't follow. What built-in functions?

Reply:
[vars()](https://docs.python.org/3.8/library/functions.html#vars)

Reply:
But it's not vars(), it's something.something.vars. What is your concern?